innovation.game.php

```diff
@@ -1700,6 +1785,12 @@ function notifyWithTwoPlayersInvolved($card, $transferInfo, $progressInfo) {
                 $message_for_others = clienttranslate('${player_name} transfers ${<}${age}${>} ${<<}${name}${>>} from his board to ${opponent_name}\'s score pile.');
                 break;
             
+            case 'board->achievements':
+                $message_for_player = clienttranslate('${You} transfer ${<}${age}${>} ${<<}${name}${>>} from your board to ${opponent_name}\'s achievements.');
+                $message_for_opponent = clienttranslate('${player_name} transfers ${<}${age}${>} ${<<}${name}${>>} from his board to ${your} achievements.');
+                $message_for_others = clienttranslate('${player_name} transfers a ${<}${age}${>} from his board to ${opponent_name}\'s achievements.');
```

Comment:
${<}${age}${>} ${<<}${name}${>>}

It's fine the say the name of the card here since the card came from a visible location.